content/chainguard/chainguard-images/getting-started/laravel.md

```diff
@@ -0,0 +1,324 @@
+      - .:/app
+    networks:
+      - wolfi
+
+  nginx:
+    image: cgr.dev/chainguard/nginx
+    restart: unless-stopped
+    ports:
+      - 8000:8080
+    volumes:
+      - .:/app
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    networks:
+      - wolfi
+
+  mariadb:
+    image: cgr.dev/chainguard/mariadb
+    restart: unless-stopped
+    environment:
+      MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 1
+      MARIADB_USER: laravel
+      MARIADB_PASSWORD: password
+      MARIADB_DATABASE: octofacts
+    ports:
+      - 3306:3306
+    networks:
+      - wolfi
+
+networks:
+  wolfi:
+    driver: bridge
+```
+
+This `docker-compose.yaml` file defines 3 services to run the application (**app**, **nginx**, **mariadb**), using volumes to share the application files within the container and a configuration file for the Nginx server, which we'll showcase in a moment. Notice the database credentials within the `mariadb` service: these environment variables are used to set up the database. This is done automatically by the MariaDB image entrypoint upon container initialization. We'll use these credentials to configure the new database within Laravel's `.env` file.
+
+>Please note this Docker Compose setup is intended for local development only. For production environments, you should never keep sensitive data like database credentials in plain text. Check the [Docker Compose Documentation](https://docs.docker.com/compose/use-secrets/) for more information on how to handle sensitive data in Compose files.
+
+The following `nginx.conf` file is also included within the root of the application folder. This file is based on the [recommended Nginx deployment configuration](https://laravel.com/docs/11.x/deployment#nginx) from official Laravel docs.
+
+```nginx configuration
+pid /var/run/nginx.pid;
+
+events {
+  worker_connections  1024;
+}
+
+http {
+    server {
+    listen 8080;
+    root /app/public;
+
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-Content-Type-Options "nosniff";
+
+    index index.php;
+
+    charset utf-8;
+
+    location / {
+    include  /etc/nginx/mime.types;
+        try_files $uri $uri/ /index.php?$query_string;
+    }
+
+    location = /favicon.ico { access_log off; log_not_found off; }
+    location = /robots.txt  { access_log off; log_not_found off; }
+
+    error_page 404 /index.php;
+
+    location ~ \.php$ {
+        try_files $uri =404;
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+        fastcgi_pass app:9000;
+        fastcgi_index index.php;
+        include fastcgi_params;
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_param PATH_INFO $fastcgi_path_info;
+    }
+
+    location ~ /\.(?!well-known).* {
+        deny all;
+    }
+  }
+}
+```
+
+This file sets up the document root to `app/public` and configures Nginx to redirect `.php` requests to the container running PHP-FPM (`app:9000`).
+
+You can bring the environment up with:
+
+```shell
+docker compose up
```

Comment:
I got this:

```
WARN[0000] /Users/amouat/tmp/edu-images-demos/php/octo-facts/docker-compose.yml: `version` is obsolete
[+] Running 2/2
 ✘ mariadb Error context canceled                                                                        1.4s
 ✘ app Error     pull access denied for octofacts, repository does not exist or may requir...            1.4s
Error response from daemon: pull access denied for octofacts, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

You can just remove the version attribute to get rid of the warning.
The error is because the octofacts is defined as `image: octofacts` in the git repo. It looks like it should have been `image: cgr.dev/chainguard/laravel:latest-dev` according to the YAML in the blog text